File: Rnwood.Smtp4dev/ApiModel/ServerRelayOptions.cs
﻿using MailKit.Security;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rnwood.Smtp4dev.ApiModel
{
    public class ServerRelayOptions
    {
        public string SmtpServer { get; set; }

        public int SmtpPort { get; set; }

        public string[] AutomaticEmails { get; set; }

        public string SenderAddress { get; set; }

        public string Login { get; set; }

        public string Password { get; set; }

        public string TlsMode { get; set;}
        public string AutomaticRelayExpression { get; set; }
    }
}

File: Rnwood.Smtp4dev/ClientApp/src/ApiClient/ServerRelayOptions.ts


export default class ServerRelayOptions {
 
    constructor(smtpServer: string, smtpPort: number, automaticEmails: string[], senderAddress: string, login: string, password: string, ) {
         
        this.smtpServer = smtpServer; 
        this.smtpPort = smtpPort; 
        this.automaticEmails = automaticEmails; 
        this.senderAddress = senderAddress; 
        this.login = login; 
        this.password = password;
    }

     
    smtpServer: string; 
    smtpPort: number; 
    automaticEmails: string[]; 
    senderAddress: string; 
    login: string; 
    password: string;
}
